### ISSUE TITLE
Add configuration parameter for number of workers. 

### ISSUE THREAD

**Issue by yaroslavNikolaev** [IzakMarais/reporter#200]
Hi there. 
I'm not a go developer. 
but can we introduce input parameter to control number of workers. 

In report.go. 
	var wg sync.WaitGroup
	workers := 5 -> replace to variable. 
	wg.Add(workers)

Reason: due to grafana works in docker with limited memory , 5 workers creates 5 phantomjs as result some of them killed. 
Thanks.

**Comment by IzakMarais** [comment]
Sounds like a good idea. Can't you just increase the memory off the docker though?

**Comment by yaroslavNikolaev** [comment]
memory = money and speed in our case is not critical at all)) 
from other hand, for someone it can be critical to perform this activity as fast as possible, additional flexibility won't hurm)

**Comment by IzakMarais** [comment]
OK, sounds reasonable. To keep the flags simpler, I would prefer something like a boolean flash that disables parallel image fetching.

**Comment by DEvil0000** [comment]
I agree, memory is a critical resource but when it takes longer it is not critical at all.
Executing some panels in parallel is nice but too many swaps or crashes a system in the worst case.
A configurable parallelization like X at a time would be nice.

**Comment by DEvil0000** [comment]
additional info: grafana has a parameter `concurrent_render_limit` but sadly it does not apply for this situation. only for alarm image rendering.
